### DISCUSSION TITLE
How can I append JSON response in useFieldArray and then add other fields?

### DISCUSSION THREAD

**Discussion by shreyanshkhare** [react-hook-form/react-hook-form#2851] [Q&A]
How can I append JSON response in useFieldArray and then add other fields?

**Comment by bluebill1049** [comment]
`reset({
  ...getValues(),
  // your data
})`

in the next release, you can use `setValue` as well.

**Reply by shreyanshkhare** [reply]
@bluebill1049  thanks for the rapid response, actually i was asking about fields and append

```
const { fields, append, remove } = useUpdateCompanyLocations({
        name: 'values'
    })
```
